src/components/Modal/ModalHeader.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { forwardRef } from 'react'
+
+import cn from 'classnames'
+
+import { IconRemove } from 'components/Icons'
+
+import styles from './ModalHeader.module.css'
+import { ModalHeaderProps, ModalTitleProps } from './types'
+
+/**
+ * Header component to be used inside modals
+ */
+export const ModalHeader = forwardRef<HTMLDivElement, ModalHeaderProps>(
+  function ModalHeader(
+    { className, onClose, showDismissButton = true, children, ...props },
+    ref
+  ) {
+    return (
+      <div
+        className={cn(styles.headerContainer, className)}
+        ref={ref}
+        {...props}
+      >
+        {showDismissButton && (
```

Comment:
i believe we are trying to do ternery instead of && for rendering! Also we might want to make this a button, though guessing we follow up since it was already a div?